### DISCUSSION TITLE
The 'missing function' problem in nidcpower needs a tweak?

### DISCUSSION THREAD

**Discussion by dbbotkin** [ni/nimi-python#1816] [Q&A]
After following all the instructions to get past the 'missing function' problem in nidcpower, the message below appeared after LabVIEW runtime 18+ was installed and reboot and shown in MAX. Sad.

![Screen Shot 2022-08-27 at 10 04 52](https://user-images.githubusercontent.com/7053580/187044147-df9b8bce-e027-4f14-807d-9f7f302219b2.png)

**Comment by marcoskirsch** [comment]
Hmmm... newer NI-DCPower versions no longer include the Soft Front Panel (it's been replaced with [InstrumentStudio](https://www.ni.com/en-us/shop/electronic-test-instrumentation/application-software-for-electronic-test-and-instrumentation-category/instrumentstudio.html)).

The fact that you got into this state is certainly a bug, but not a bug in the `nidcpower` Python module. Can you confirm what version of NI-DCPower you have installed, and whether you were attempting to upgrade or downgrade to a different version and what version that would be?

Try to install the LabVIEW Runtime Engine 2018 using NI Package Manager. I believe that will get the Soft Front Panel to function again.

**Comment by dbbotkin** [comment]
I like the Instrument Studio. And yes, I was attempting to upgrade to see if I could get the Python working with DCPower. I was all the way up to 2020 without problems (until I did a reboot and am now recovering after removing all NI apps in the proper order. Now I am installing 18.1 with whatever else comes along and the boot problem is gone. 

One thing that might help explain is that I installed the IVI Compliance software thinking that might be needed. I suspect that could be the source of driver issues?

Cheers,
Donal
